2-mandatory/2-financial-times.js

```diff
@@ -23,15 +37,39 @@ function titleWithFewestWords(allArticleTitles) {
     (Hint: remember that you can also loop through the characters of a string if you need to)
 */
 function headlinesWithNumbers(allArticleTitles) {
-    // TODO
+    let clickables = [];
+
+    for (const title of allArticleTitles){
+        for (const character of title){
+            if ('0123456789'.includes(character)){ //if (character = Number) my previous -failed- solution.
+                clickables.push(title);
+               break;
+            }
+            
+        }
+    }
+    return clickables;
 }
 
 /*
     The Financial Times wants to understand what the average number of characters in an article title is.
     Implement the function below to return this number - rounded to the nearest integer.
 */
 function averageNumberOfCharacters(allArticleTitles) {
-    // TODO
+    let charactersCount= 0;
```

Comment:
Keep the styling consistent. Setting up your IDE properly can help enforce that including indentation and other styling suggestions

```suggestion
    let charactersCount = 0;
```